servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractTestRest.java

```diff
@@ -1897,6 +1906,7 @@ private void verifyMetadataProperties(Tag tag) throws NessieNotFoundException {
     assertThat(referenceMetadata.numCommitsBehind()).isNull();
     assertThat(referenceMetadata.commitMetaOfHEAD()).isEqualTo(commitMeta);
     assertThat(referenceMetadata.commonAncestorHash()).isNull();
+    assertThat(referenceMetadata.numTotalCommits()).isEqualTo(10);
```

Comment:
hm this should only be set if `fetchAdditionalInformation` was set to `true`, otherwise we'll be breaking the Python client (since it doesn't know anything about `ReferenceMetadata` yet.